.github/workflows/bump_external_releases.py

```diff
@@ -68,6 +68,13 @@ def bump_google_java_formatter():
     )
 
 
+def bump_palantir_java_formatter():
+    return bump_release(
+        github_project="jsonschema2dataclass/palantir-cli",
+        tool_name="palantir-cli",
+    )
+
+
 if __name__ == "__main__":
     something_is_bumped = False
     for bumper in (bump_ktfmt, bump_ktlint, bump_google_java_formatter):
```

Comment:
You are missing `bump_palantir_java_formatter` here. Instead of continuing to update this line, you could automatically call all bumping functions as follows:

```py
something_is_bumped = False
for name, obj in tuple(locals().items()):
    if callable(obj) and name.startswith("bump") and name != "bump_release":
        something_is_bumped |= obj()
```

Reply:
done

Reply:
I'll follow Zen Of Python (_"Explicit is better than implicit"_)

I'd leave this refactoring to other PR, as I personally don't agree with querying locals. it may lead to problems which hard to test.

Reply:
No problem!